axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/AxonServerEventStore.java

```diff
@@ -259,6 +261,18 @@ public Builder eventSerializer(Serializer eventSerializer) {
             return this;
         }
 
+        /**
+         * Sets the default context for this event store to connect to.
+         *
+         * @param defaultContext for this event store to connect to.
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public Builder defaultContext(String defaultContext) {
+            assertNonNull(defaultContext, "The default context may not be null");
```

Comment:
I think it would be better if we use `BuilderUtils#assertNonEmpty`. This method will also validate that the given `defaultContext` isn't an empty `String`.